pkg/serverless/trace/inferredspan/inferred_span.go

```diff
@@ -0,0 +1,71 @@
+package inferredspan
+
+import (
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/serverless/tags"
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	// tagInferredSpanTagSource is the key to the meta tag
+	// that lets us know whether this span should inherit its tags.
+	// Expected options are "lambda" and "self"
+	tagInferredSpanTagSource = "_inferred_span.tag_source"
+
+	// additional function specific tag keys to ignore
+	functionVersionTagKey = "function_version"
+	coldStartTagKey       = "cold_start"
+)
+
+var functionTagsToIgnore = [...]string{
```

Comment:
I think including the `...` in between the `[]` before `string` means that this is now an array with length 11 instead of a slice.

Here is a blog post I found covering some of the differences: https://medium.com/@marty.stepien/arrays-vs-slices-bonanza-in-golang-fa8d32cd2b7c

tl;dr Slices are more flexible than arrays and I would always use a slice unless I had some special reason to use an array.